pkgs/development/libraries/dav1d/default.nix

```diff
@@ -31,7 +31,7 @@ stdenv.mkDerivation rec {
     "-Denable_examples=${lib.boolToString withExamples}"
   ];
 
-  doCheck = true;
+  doCheck = stdenv.hostPlatform == stdenv.buildPlatform;
```

Comment:
in `buildPythonPackage` we could also convert `true` to `stdenv.hostPlatform == stdenv.buildPlatform;`. I highly doubt we're going to see the use of an emulator in the `checkPhase` anytime soon.

Reply:
If you don't know of many package whose checks can run on the buildPlatform, sure. Just revert my patch again in your PR then.